source/user-manual/capabilities/system-inventory/viewing-system-inventory-data.rst

```diff
@@ -321,7 +321,7 @@ Follow the steps below to query the system inventory indices from the command li
                 "agent" : {
                   "id" : "006",
                   "name" : "Cent_Stream",
-                  "version" : "v4.13.0"
+                  "version" : "v|WAZUH_CURRENT|"
```

Comment:
We better don't use variables in command outputs because it should reflect the actual output not what we suppose it should be